dashboard/app/assets/stylesheets/application.css.scss

```diff
@@ -2501,10 +2501,10 @@ $width: 400px;
   position: fixed;
   top: 0;
   right: 0;
-  color: white;
+  color: $white;
   background-color: $light_purple;
   padding: 3px;
   z-index: 100;
   cursor: pointer;
-  box-shadow: 0 0 10px 0 rgba(0, 0, 0, 0.75);
+  box-shadow: 0 0 7px 2px $shadow;
```

Comment:
The definition of `$shadow` below is `rgba(0, 0, 0, 0.3);`. Is this change intentional?

Also, is the change from `0 0 10px 0` to `0 0 7px 2px` intentional?